rocAL/include/loaders/circular_buffer.h

```diff
@@ -37,6 +37,9 @@ struct DecodedDataInfo {
     std::vector<uint32_t> _roi_height;
     std::vector<uint32_t> _original_width;
     std::vector<uint32_t> _original_height;
+    std::vector<uint32_t> _original_audio_samples; //! Amplitude of an audio signal at a specific point in time
```

Comment:
if we can combine all the audio data into single tensor, these vectors can be combined

Reply:
Please let me know if we can introduce a struct for audio, which can have these 3 vectors specific to audio, and we can add an instance of the struct in this DecodeDataInfo struct